pkgs/tools/system/ac/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, ats }:
+
+stdenv.mkDerivation rec {
+  name = "ac-0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "vmchale";
+    repo = "fastcat";
+    rev = "5031c47631ea5a31d4eb8a584c8226a698453125";
```

Comment:
Can you also tag this release on github, so we can use the version tag here instead of the commit.